awx_collection/plugins/modules/project.py

```diff
@@ -387,7 +389,8 @@ def main():
         # this is resolved earlier, so save an API call and don't do it again in the loop above
         project_fields['organization'] = org_id
 
-    if scm_type == '' and local_path is not None:
+    # Respect local_path if scm_type is manual type or not specified
+    if scm_type in ('', None) and local_path is not None:
```

Comment:
I am sick and not thinking clearly, but I think this change makes sense. If it's the empty string, it was given by the user as "manual" per above. If it's `None`, it wasn't given by the user, and (I think) the API will default to "" (manual) if it's a new project.

It looks like in the serializer we already correctly error the case where the user could specify `local_path` for an SCM type where it doesn't make sense. Still, there are probably some test case scenarios it would be good to add here in the collection just to be explicit. Not :+1:-blocking though.